templates/single.twig

```diff
@@ -4,27 +4,66 @@
 	<div class="content-wrapper">
 		<article class="post-type-{{ post.post_type }}" id="post-{{ post.ID }}">
 			<section class="article-content">
+				<div><span>{{ __('POST') }}</span>
```

Comment:
Needs domain specified, why is it in all caps? I'd thought that kind of styling is done in CSS?